CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 * [CHANGE] Remove middleware package. #182
 * [CHANGE] Memberlist: disabled TCP-based ping fallback, because dskit already uses a custom transport based on TCP. #194
 * [CHANGE] Memberlist: KV store now fast-joins memberlist cluster before serving any KV requests. #195
+* [CHANGE] Ring: remove duplicate state in NewOp func #203
```

Comment:
this looks like an accidental add

Reply:
It has been reordered from below. This change is good 👍